src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -108,8 +108,11 @@
   
     <!-- Publish capability enables the Publish command for the Project -->
     <ProjectCapability Include="Publish"/>
-  </ItemGroup>
 
+    <!-- All Microsoft.NET.Sdk Exe projects, except WPF and WinForms exes, can be published to app service -->
+    <ProjectCapability Include="AppServicePublish" Condition="'$(OutputType)' == 'Exe'" />
```

Comment:
I think maybe we should put FolderPublish here for consistency even though it duplicates sdk targets. We could then remove from sdk and have justone place to maintain.

Did we get confirmation that output type == exe (but not winexe) is the right trigger for all the cases of app service publish? I believe it is right for all the sdk cases, but I'm not the authority on it.

Reply:
> I think maybe we should put FolderPublish here

I agree, but wasn't sure if duplicates are a problem. Will add.

> Did we get confirmation...

I assumed that you guys had worked out the details for this offline. Apologies if I jumped at fixing it too soon, was aiming for Preview 2.

Reply:
> We could then remove from sdk and have just one place to maintain.

https://github.com/dotnet/sdk/issues/2769